lambda/handlers/intentHandler.js

```
@@ -540,6 +558,152 @@ const DeleteDataIntentHandler = {
+      Alexa.getRequestType(handlerInput.requestEnvelope) === "IntentRequest" &&
+      Alexa.getIntentName(handlerInput.requestEnvelope) === "HadithIntent"
+    );
+  },
+  async handle(handlerInput) {
+    console.log("In HadithIntentHandler");
+    const requestAttributes = handlerInput.attributesManager.getRequestAttributes();
+    const locale = helperFunctions.splitLanguage(Alexa.getLocale(handlerInput.requestEnvelope));
+    const hadith = await getRandomHadith(locale).catch((error) => {{
+      console.log("Error in fetching hadith: ", error);
+      return requestAttributes.t("hadithErrorPrompt");
+    }});
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)